cmd/containerboot/main.go

```diff
@@ -352,6 +364,44 @@ runLoop:
 				addrs := n.NetMap.SelfNode.Addresses().AsSlice()
 				newCurrentIPs := deephash.Hash(&addrs)
 				ipsHaveChanged := newCurrentIPs != currentIPs
+
+				if cfg.TailnetTargetFQDN != "" {
+					var (
+						egressAddrs          []netip.Prefix
+						newCurentEgressIPs   deephash.Sum
```

Comment:
Perhaps `newEgressIPs`?

Reply:
It's currentEgressIPs, but before the old value has been used in a check. This was broken, I've fixed it now, so hopefully the var name makes more sense